README.md

```diff
@@ -0,0 +1,55 @@
+# My Country Documentation with Microverse
+
+> Description the project.
+
+## Built With
+
+- HTML
+- CSS
+- GIT
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/shihabrafio)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Have git installed on your local machine/PC.
+
+### Setup
+
+Open the terminal and run `git clone https://github.com/shihabrafio/Code-Review.git`
+
+### Usage
+
+Right-click on the index.html file and open with any browser.
+
+### Deployment
+
+You can push your project to github using `git add .` , `git commit -m "message"` and `git push`
+
+## Authors
+
+👤 **Shihab Uddin Khan**
+
+- GitHub: [@shihabrafio](https://github.com/shihabrafio)
+- Twitter: [@shihab_uddin19](https://twitter.com/shihab_uddin19)
+- LinkedIn: [Shihab Uddin Khan](https://www.linkedin.com/in/shihab-uddin-khan-45620a16a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/).
```

Comment:
>  * [ ]  Kindly also remember to update the link to your project issues page. It is normally
>   `https://github/projectname/issues`. Because `https://github.com/` opens up the GitHub main page , not the project. As you are asking to check the project, you need to specify the project.